src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const images = [...document.querySelectorAll('a')];
+
+images.forEach((image) => {
+  image.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const link = e.target.closest('img');
+
+    if (!link) {
+      return;
+    }
+
+    const large = document.getElementById('largeImg');
+
+    large.src = image.href;
```

Comment:
The `large.src = image.href;` line is incorrect because `image` is an anchor element, not an image. You should use `link.href` instead, assuming `link` is correctly set to the anchor element.